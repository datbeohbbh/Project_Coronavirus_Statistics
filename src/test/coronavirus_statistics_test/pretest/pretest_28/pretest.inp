23 25
q 1 9 431
q 1 6 413
? 6
? 4 2
? 6
q 1 8 436
? 6
q 2 9 428
q 1 4 444
q 1 18 95
q 1 13 127
q 2 13 55
q 2 8 196
? 7
q 2 8 177
q 3 18 56
q 1 5 111
q 1 16 225
? 4 2
? 6
? 7
? 6
? 6
? 4 5
? 4 5