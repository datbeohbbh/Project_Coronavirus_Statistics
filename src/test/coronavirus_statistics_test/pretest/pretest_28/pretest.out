Query ? 6 :
Currently, no recovery cases have been recorded

Query ? 4 2 :
The answer is the city with the index 9 
- Information of the city 9 -
-> Number of current patients : 431
-> Has recovered : 0
-> Was dead : 0

Query ? 6 :
Currently, no recovery cases have been recorded

Query ? 6 :
Currently, no recovery cases have been recorded

Query ? 7 :
Currently, no deaths have been recorded

Query ? 4 2 :
The answer is the city with the index 18 
- Information of the city 18 -
-> Number of current patients : 39
-> Has recovered : 0
-> Was dead : 56

Query ? 6 :
The city with the highest recovery is city with the index 9
Quantity: 428 people

Query ? 7 :
The city with the highest death is city with the index 18
Quantity: 56 people

Query ? 6 :
The city with the highest recovery is city with the index 9
Quantity: 428 people

Query ? 6 :
The city with the highest recovery is city with the index 9
Quantity: 428 people

Query ? 4 5 :
The answer is the city with the index 5 
- Information of the city 5 -
-> Number of current patients : 111
-> Has recovered : 0
-> Was dead : 0

Query ? 4 5 :
The answer is the city with the index 5 
- Information of the city 5 -
-> Number of current patients : 111
-> Has recovered : 0
-> Was dead : 0

